### DISCUSSION TITLE
How to change Report email's text?

### DISCUSSION THREAD

**Discussion by aortiz-WW** [apache/superset#22344] [Q&A / Help] [answered]
I am running [apache superset 2.0.0-dev](https://github.com/apache/superset). Everything works great. I just want to change the text that comes inside the report emails. Here is a photo of the current email it sends (this is just some example dashboard that superset provides): 
![image](https://user-images.githubusercontent.com/118770354/205961991-ab2c2ae0-5691-4a95-b162-3a407872bae8.png)

The text in the hyperlink that I circle is what I want to change. I have already gone through a lot of the code looking for that string "Explore in Superset". As well as searched through the [repo on github](https://github.com/apache/superset/search?q=%22Explore%20in%20Superset%22&type=code)

I changed every instance I can find of the string "Explore in Superset" to the text I want, but none have an effect on the email that is sent.

**Comment by rusackas** [comment]
@yousoph sending this your way so that perhaps we can better decide as to treat this as a feature request (i.e. making this configurable) or providing further instruction as to how to customize this. It does indeed look like [this line](https://github.com/apache/superset/blob/8afe724e9e69f2b2da55fa4d60d65d51641ffce9/superset/reports/notifications/email.py#L131) is the one that would need tweaking.

**Comment by sfirke** [comment, marked as answer]
Looks like this just got implemented on the main branch: https://github.com/apache/superset/pull/19779
